website/docs/reference/commands/compile.md

```diff
@@ -11,4 +11,22 @@ The `compile` command is useful for:
 2. Manually running compiled SQL. While debugging a model or schema test, it's often useful to execute the underlying `select` statement to find the source of the bug.
 3. Compiling `analysis` files. Read more about analysis files [here](/docs/build/analyses).
 
-It is _not_ a pre-requisite of `dbt run`.
+<VersionBlock firstVersion="1.5">
```

Comment:
We deprecated the use of changelogs in favor of version blocks to help reign in the amount of information displayed on the screen. Since we version the docs themselves, this makes a generally more pleasing UX, and gives us a nice way to dynamically change the text (where appropriate) for different versions of the product.